### DISCUSSION TITLE
Nokia SROS issue with return value in ["xx"]

### DISCUSSION THREAD

**Discussion by theblues25** [ktbyers/netmiko#2458] [General]
I used netmiko with nokia sros (md-cli) and when i use send_command to show something and return value has some text in a bracket like this ["xxx"] . the return value disappears. below is some example

when i show some command by myself (ssh to node)
    /configure policy-options policy-statement "vrf-import" entry 10 from protocol name [bgp-vpn]
    /configure policy-options policy-statement "vrf-import" entry 10 action action-type accept

when i got return value of send_cmd of netmiko
    /configure policy-options policy-statement "vrf-import" entry 10 from protocol name     /configure policy-options policy-statement "vrf-import" entry 10 action action-type accept

i already try to use device_type='nokia_sros' and 'alcatel_sros" but result is same

**Comment by scottweese** [comment]
I am having this issue as well (specifically with "/admin show configuration json" command) .  Any fix you've found?
